recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ about:
   home: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-file-datalake
   license: MIT
   license_family: MIT
-  license_file: LICENSE.txt
+  license_file: LICENSE
```

Comment:
***xref:*** https://github.com/Azure/azure-sdk-for-python/pull/21962#discussion_r760607651